website/docs/r/schema_template_vrf.html.markdown

```diff
@@ -34,7 +36,8 @@ resource "mso_schema_template_vrf" "vrf1" {
 * `display_name` - (Required) The name as displayed on the MSO web interface.
 * `layer3_multicast` - (Optional) Whether to enable L3 multicast.
 * `vzany` - (Optional) Whether to enable vzany.
-
+* `ip_data_plane_learning` - (Optional) Whether IP data plane learning is enabled or disabled. Allowed values are `disabled`and `enabled`. Default to `enabled`.
+* `preferred_group` - (Optional) Whether to enable preferred Endpoint Group.
```

Comment:
Allows enabling the preferred Endpoint Group

Reply:
Since all the previous arguments are explained that way, changing explanation for one argument is not feasible.